gradle/libs.versions.toml

```diff
@@ -0,0 +1,141 @@
+[versions]
+# Build tools and plugins
+gradle-release = "3.0.2"
+spotbugs = "6.0.28"
+spotless = "6.23.3"
+kotlin = "1.9.25"
+spring-dependency-management = "1.1.7"
+spring-boot = "3.1.12"
```

Comment:
shall we move tthe spring and kotlin versions to examples?

I am actually considering moving the examples to a different repo, not now, but i have the feeling that is starting to be too much in this repo, especially when adding other crypto like vault....

Reply:
Done!